docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -92,49 +277,78 @@ See [provider contracts](../contracts/overview.md) for more details.
   - `MachinePoolPhase`, used in `machinePool.status.phase`
   - `MachinePhase`, used in `machine.status.phase`
   
-### Removals
+## Removals
 
 There are no removals in the CAPI v1.11 version.
```

Comment:
I think this is not correct anymore given all the removals we mentioned above. I think I would just integrate what we have in this section into the API changes section above

Reply:
changes the paragraph scope a little, PTAL